state_machine/applications/system-check/tests/burnwire_test.py

```diff
@@ -0,0 +1,86 @@
+"""
+Python system check script for PyCubed satellite board
+PyCubed Mini mainboard-v02 for Pocketqube Mission
+Burnwire Deployment Test
+* Author(s): Yashika Batra
+"""
+
+import time
+from lib import pycubed as cubesat
+
+
+def user_test():
```

Comment:
This function name isn't very descriptive, could we please change it to something more reflective of what it does.